### DISCUSSION TITLE
TEST: Who sees this?

### DISCUSSION THREAD

**Discussion by johnbent** [tekinged/missing#59] [Q&A]
Please reply if you see this. This is a way for us to discuss other tekinged issues not related to missing words. Things like, "A quiz is broken" or, "it'd be cool to add music" or stuff like that.

**Comment by Gnalu** [comment]
Ng kora kmal ngii…langsu.

**Comment by s-o-u-a-n-g** [comment]
Ngak. Ak mesang. 

souang
